src/htmlParser.js

```diff
@@ -32,12 +32,16 @@ class HtmlParser {
 
       const contentText = comment.contentText.runs
 
+      let customEmojis = []
```

Comment:
If you don't reassign value you can/should use `const`

Reply:
I always forget to do this with arrays. Good catch